os/arch/arm/src/imxrt/imxrt_mpuinit.c

```diff
@@ -150,7 +159,7 @@ void imxrt_mpu_initialize(void)
 #ifdef CONFIG_BUILD_PROTECTED
 void imxrt_mpu_uheap(uintptr_t start, size_t size)
```

Comment:
This API will no longer be required. Please delete the entire API.

Reply:
deleted